.gitignore

```diff
@@ -235,8 +235,7 @@ spack-*
 install/
 ## podio
 podio_generated_files.cmake
-/include/podio/podioVersion.h
-/python/podio/__init__.py
```

Comment:
These come from https://github.com/key4hep/key4hep-dev-utils/blob/main/defaults/.gitignore#L238, I think copied and pasted from the `.gitignore` in podio. The up to date ones seem to be 
```
/python/podio/__version__.py
/python/podio_version.py
```

Reply:
Ah, this is the global .gitignore, right? Why are we having podio specific ignores there? These should not appear in any repository, and the podio .gitignore is not globally managed.

Reply:
Ok they should be removed now, the changes here can be kept or not, they will be overwritten at some point anyway